aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -1019,26 +1032,54 @@ module aptos_framework::fungible_asset {
         assert!(amount == 0, error::invalid_argument(EAMOUNT_IS_NOT_ZERO));
     }
 
-    public(friend) fun deposit_internal(store_addr: address, fa: FungibleAsset) acquires FungibleStore, ConcurrentFungibleBalance {
+    inline fun raw_deposit_internal(
```

Comment:
Naming here seems pretty bad. Both `raw` and `internal` imply the same thing? I think what we are really referring here is `unchecked_deposit_with_no_events`?